itchiodl/game.py

```diff
@@ -64,8 +64,11 @@ def load_downloads(self, token):
                 headers={"Authorization": token},
             )
         j = r.json()
-        for d in j["uploads"]:
-            self.downloads.append(d)
+        # This converts a null j.uploads into an empty dictionary and uses the safer .get accessor
+        if "uploads" in j:
+            j.update({"uploads": []})
+        for uploads in j.get("uploads"):
```

Comment:
can be indented into the if and use [] accessor

Reply:
Should be fixed now